_pieces_lib/pieces_os_client/wrapper/websockets/health_ws.py

```diff
@@ -1,9 +1,22 @@
 from .base_websocket import BaseWebsocket
 
 class HealthWS(BaseWebsocket):
+	def __init__(self, pieces_client, 
+		on_open_callback = None):
+		self.is_loaded = False
+		super().__init__(pieces_client, lambda x: None, on_open_callback, None,self.on_close)
+	
 	@property
 	def url(self):
 		return self.pieces_client.HEALTH_WS_URL
 
 	def on_message(self, ws, message):
-		self.on_message_callback(message)
\ No newline at end of file
+
+		if "OK" in message.lower():
```

Comment:
We are checking here if uppercase "OK" is in the string after we convert it to lowercase